SwiftCheck/Check.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Check.swift
+//  SwiftCheck
+//
+//  Created by Robert Widmann on 1/19/15.
+//  Copyright (c) 2015 Robert Widmann. All rights reserved.
+//
+
+import Basis
+
+public var swiftCheck : QuickCheck = QuickCheck()
```

Comment:
Now that I think about it, `property` is a better name for this.  Matches ScalaCheck better.